<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="img_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>191</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="info_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>191</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pre_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>上一个</string>
    </property>
   </widget>
   <widget class="QPushButton" name="next_btn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>530</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>下一个</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>256</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="onlineUser_btn">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>在线用户</string>
    </property>
   </widget>
   <widget class="QPushButton" name="regUser_btn">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>70</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>注册用户</string>
    </property>
   </widget>
   <widget class="QLabel" name="show_label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="back_btn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>842</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
